src/account/operations/output_consolidation.rs

```diff
@@ -1,13 +1,24 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use iota_client::block::output::{
-    unlock_condition::{AddressUnlockCondition, UnlockCondition},
-    BasicOutputBuilder, NativeTokensBuilder, Output,
+use iota_client::block::{
+    input::INPUT_COUNT_MAX,
+    output::{
+        unlock_condition::{AddressUnlockCondition, UnlockCondition},
+        BasicOutputBuilder, NativeTokensBuilder, Output,
+    },
 };
 #[cfg(feature = "ledger_nano")]
 use iota_client::secret::SecretManager;
 
+// Constants for the calculation of the amount of inputs we can use with a ledger nano
+#[cfg(feature = "ledger_nano")]
+const ESSENCE_SIZE_WITHOUT_IN_AND_OUTPUTS: usize = 49;
+#[cfg(feature = "ledger_nano")]
+const INPUT_SIZE_IN_ESSENCE: usize = 43;
```

Comment:
How do you get 43 ?

Reply:
Got the number from Thomas
```Rust
    let input = Input::Utxo(UtxoInput::from_str("0xb66fd384cb5755668f1890ea2e41d699db9cf32f3bc422ad3c24ffeb9c7f01d00000").unwrap());
    println!("{}", input.packed_len());
```
prints 35, so I guess this is the correct number to use?

Reply:
Got the number and din't think further about it :trollface: 
It's 35 + 8 for the bip32 path part we need in addition to it, will add a comment and change the name

Reply:
Ah I see, wasn't accounting for the bip32 path, thanks